src/js/control-bar/audio-track-controls/audio-track-menu-item.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * @file audio-track-menu-item.js
+ */
+import MenuItem from '../../menu/menu-item.js';
+import Component from '../../component.js';
+import * as Fn from '../../utils/fn.js';
+
+/**
+ * The audio track menu item
+ *
+ * @param {Player|Object} player
+ * @param {Object=} options
+ * @extends MenuItem
+ * @class AudioTrackMenuItem
+ */
+class AudioTrackMenuItem extends MenuItem {
+  constructor(player, options) {
+    let track = options.track;
+    let tracks = player.audioTracks();
+
+    // Modify options for parent MenuItem class's init.
+    options.label = track.label || track.language || 'Unknown';
+    options.selected = track.enabled;
```

Comment:
I'm not following, AudioTrackButton passes `selectable: true` to populate the menu with clickable menu items. It also passes track which has an `enabled` property which is really the currently active track. If it is active we want to indicate that it is selected by setting AudioTrackMenuItem's `selected` option.